src/inline-assembly.md

```diff
@@ -1346,6 +1379,13 @@ r[asm.rules.preserved-registers]
     - Vector extension state (`vtype`, `vl`, `vxsat`, and `vxrm`).
   - LoongArch
     - Floating-point condition flags in `$fcc[0-7]`.
+  - PowerPC/PowerPC64
+    - Floating-point status and sticky bits in the `fpscr` (any field other than DRN, VE, OE, UE, ZE, XE, NI, or RN).
+    - Vector status and sticky bits in the `vscr` (any field other than NJ).
+    - The `mq` register (this register only exists on POWER1 and POWER2).
+  - PowerPC SPE
+    - The sticky and status bits of the `spefscr` (any field other than FINXE, FINVE, FDBZE, FUNFE, FOVFE, or FRMC)
+    - The Accumulator register `acc` (LLVM names this `spe_acc`).
```

Comment:
This should probably be defined as a clobber-only register. This should be fine to stabilize along with the other registers here.